packages/notifi-frontend-client/README.md

```diff
@@ -1,11 +1,238 @@
+```
+
+## 🧩 Build Your UI Components
+
+Common patterns for UI involve rendering a form in a modal or card to collect the user's contact information (Email, SMS, Telegram, etc).
+
+## 🪝 Hook up the SDK 
+
+Load the Notifi Frontend Client SDK into your component.
+
+```
+const { newAptosClient, newAptosConfig } = require('@notifi-network/notifi-frontend-client');
+```
+
+Instantiate and configure the Notifi Client for your dApp and environment. If your user has not connected their wallet, they will need to do so in order to instantiate the client.
+
+```
+const tenantId = <Tenant ID received through the Notifi Config Tool>;
+const account = <Your Martian account metadata>;
+const blockchainEnv = "Development";
+
+const config = newAptosConfig(account, tenantId, blockchainEnv);
+const client = newAptosClient(config);
+
+```
+
+## 🔏 Signature Authorization
+
+For a user to opt-in for notifications, they will need to provide their signature. This signature will then be used to authorize the user's connected wallet address with Notifi and create the account with Notifi.
+
+Using the wallet adapter of your choice, prompt the user to sign and use the signed message in the `logIn()` hook.
+
+```
+const martianLogIn = async () => {
+    const loginResult = await client.logIn({
+        walletBlockchain: 'APTOS',
+        signMessage: async (message, timestamp) => {
+            const signedMessage = await wallet.signMessage({
+                address: true,
+                application: false,
+                chainId: false,
+                message,
+                nonce: timestamp,
+            });
+            console.log('signedMessage', signedMessage);
+            return signedMessage.signature;
+        },
+      });
+      // client should be authenticated now
+      console.log('loginResult', loginResult);
+      return loginResult;
+};
+
+```
+
+## 🕹 Rendering Alert Options
+
+After the user successfully authorizes, fetch the newly created user data from Notifi using the `getSourceGroups()` hook.
+
+In our simplest use case, the user will have 1 entry in the `sources` array, which will be based on their connected wallet address.
+
+```
+const sourceData = await client.getSourceGroups();
+
+// An array of sources that belong to the user
+const { sources } = sourceData;
 ```
-const notifiFrontendClient = require('@notifi-network/notifi-frontend-client');
 
-// TODO: DEMONSTRATE API
+You'll want to render the alert options available for the user's source, based on what is returned in the source's `applicableFilters` array. More about the [`Filter` type here](https://notifi-network.github.io/notifi-sdk-ts/modules.html#Filter).
+
+```
+// Render the options to the user
+const {id, applicableFilters} = sources?.[0];
+const filterId = applicableFilters?.[0].id;
+```
+
+For more complex scenarios where the user has multiple sources, you may want to iterate over each source to accumulate the applicable filters to render.
```

Comment:
this is probably not useful since the API to ensureSourceGroup already expects multiple sources (so it's already the "more complex" scenario)